### PR TITLE
Added flowdock scanner

### PR DESCRIPTION
**Pull request by roxanne-tampus**

### CODE REVIEW

File: pkg/detectors/flowdock/flowdock.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"flowdock"}) + `\b([0-9a-z]{32})\b`)
```
Comment (ahrav) [inline pkg/detectors/flowdock/flowdock.go:23]: suggestion: this regex can be made a bit stricter `\b([0-9a-f]{32})\b`

File: pkg/detectors/flowdock/flowdock.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+		}
+		resMatch := strings.TrimSpace(match[1])
+
+		s1 := detectors.Result{
+			DetectorType: detectorspb.DetectorType_Flowdock,
+			Raw:          []byte(resMatch),
+		}
+
+		if verify {
+			req, err := http.NewRequestWithContext(ctx, "GET", "https://api.flowdock.com/flows", nil)
```
Comment (ahrav) [inline pkg/detectors/flowdock/flowdock.go:50]: nit: Maybe use `https://api.flowdock.com/flows?users=false` in their documentation they state:

`A boolean value (1/0) that controls whether a list of users should be included with each flow.` 
Based on that wording (1 before 0) that indicates to me the default behavior is to include users within the response if that is the case we should probably pass in `users=false` to return the response object size. If that is not the case feel free to ignore this 😄

**Review by ahrav — APPROVED** [review]
couple suggestions otherwise lgtm 👍

File: pkg/detectors/flowdock/flowdock.go (hunk 2 above)
Comment (roxanne-tampus) [reply pkg/detectors/flowdock/flowdock.go]: Sure, I just pushed the changes. Thanks @ahrav!

File: pkg/detectors/flowdock/flowdock.go (hunk 1 above)
Comment (dustin-decker) [reply pkg/detectors/flowdock/flowdock.go]: Some extra context: it's hex encoding that uses the a-f characters rather than a-z

**Pull request merged**